### ISSUE TITLE
Training vs Validation loss

### ISSUE THREAD

**Issue by amehrjou** [benediktdietz/EHRanalysis#4]
I ran the this experiment with num_patients=1000. The training loss is larger than the validation loss.

![image](https://user-images.githubusercontent.com/17348736/92906405-bc195c00-f424-11ea-916f-f3709f862989.png)

https://github.com/benediktdietz/EHRanalysis/blob/32f152c54790d0f69c6d65969421fd478f8af161/ehr_analyis.py#L35
